lib/langchain/llm/ollama.rb

```diff
@@ -265,5 +261,40 @@ def client
         conn.response :raise_error
       end
     end
+
+    def json_responses_chunk_handler(&block)
+      proc do |chunk, _size|
+        chunk.split("\n").each do |chunk_line|
+          parsed_chunk = begin
+            JSON.parse(chunk_line)
+          rescue JSON::ParserError
+            # In some cases the chunk exceeds the buffer size and the JSON parser fails.
+            # TODO: How to better handle this?
```

Comment:
it's not clear what we should do here @andreibondarev . wdyt?

Reply:
I wonder if the error just needs to be thrown? There's no graceful way to recover from this and still provide a useful response, I don't think?

Reply:
yeah, let's remove that and let it blow up, until we figure out a way to handle this.